README.md

```diff
@@ -15,12 +15,29 @@ NCEPLIBS inquiries, contact [Edward
 Hartnett](mailto:edward.hartnett@noaa.gov) (secondary point of contact
 [Alex Richert](mailto:alexander.richert@noaa.gov)).
 
-## Related NCEPLIBS Projects
+## Prerequisite External Projects
+
+Project | Notes
+-----------|------
+[libjasper.a](http://www.ece.uvic.ca/~mdadams/jasper/) | [JPEG-2000](http://www.jpeg.org/JPEG2000.html) library.
```

Comment:
Unless it specifically has to be static jasper, then I'd change libjasper.a -> Jasper

Reply:
Good point!